compute_tools/src/spec_apply.rs

```diff
@@ -419,7 +419,7 @@ impl ComputeNode {
                 .iter()
                 .filter_map(|val| val.parse::<usize>().ok())
                 .map(|val| if val > 1 { val - 1 } else { 1 })
-                .last()
+                .next_back()
```

Comment:
IIUC, `next_back` is just a more efficient version of `last`, i.e. doesn't consume the whole iterator, right?

Reply:
there is some scenarios where `next_back` is faster than `last`, because someone thought about implementing it efficiently while they didn't implement `last` efficiently. Personally I think it's a bug of iterator writers and that in those instances, one should also implement `last` efficiently. Also, `last` reads nicer than `next_back`.

But shrug, now we have a lint for this and I don't we should start a list of allowed clippy lints for this.